pnpbridge/quickstart.md

```diff
@@ -46,73 +105,43 @@ After cloning the Azure PnP Bridge repo to your machine, open the "Developer Com
   }
   ```
-  Or using DPS:
 
-  ```JSON
-  {
-      "connection_parameters": {
-        "connection_type" : "dps",
-        "device_capability_model_uri": "[To fill in]",
-        "auth_parameters" : {
-          "auth_type" : "symmetric_key",
-          "symmetric_key" : "[DEVICE KEY]"
-        },
-        "dps_parameters" : {
-          "global_prov_uri" : "[GLOBAL PROVISIONING URI]",
-          "id_scope": "[IOT HUB ID SCOPE]",
-          "device_id": "[DEVICE ID]"
-        }
-      }
-    }
-  }
-  ```
-  > Note: Refer to the [Azure IoT Central documentation on device connectivity](https://docs.microsoft.com/azure/iot-central/concepts-connectivity-pnp?toc=/azure/iot-central-pnp/toc.json&bc=/azure/iot-central-pnp/breadcrumb/toc.json) for how to generate the id_scope, device_id, and symmetric_key for your device. The device_capability_model_uri is the "CapabilityModelId" that is listed for your device's Device Capability Model in Azure IoT Central.
  
  Once filled in you the config file should resemble:
    ```JSON
     {
       "connection_parameters": {
         "connection_type" : "connection_string",
         "connection_string" : "[CONNECTION STRING]",
-        "device_capability_model_uri": "urn:YOUR_COMPANY_NAME:sample_device:1",
+        "root_interface_model_id": "dtmi:com:example:SampleDevice;1",
         "auth_parameters" : {
           "auth_type" : "symmetric_key",
         }
       }
     }
   }
   ```
-  Or using DPS:
 
-  ```JSON
-  {
-      "connection_parameters": {
-        "connection_type" : "dps",
-        "device_capability_model_uri": "urn:YOUR_COMPANY_NAME:sample_device:1",
-        "auth_parameters" : {
-          "auth_type" : "symmetric_key",
-          "symmetric_key" : "[DEVICE KEY]"
-        },
-        "dps_parameters" : {
-          "global_prov_uri" : "[GLOBAL PROVISIONING URI] - typically it is global.azure-devices-provisioning.net",
-          "id_scope": "[IOT HUB ID SCOPE]",
-          "device_id": "[DEVICE ID]"
-        }
-      }
-    }
-  }
-  ```
+## Get the required dependencies (Environmental Sensor)
+
+### Download the models
+
+Later, you use Azure IoT explorer to view the device when it connects to your IoT hub. Azure IoT explorer needs a local copy of the model file that matches the **Model ID** your device sends. The model file lets Azure IoT explorer display the telemetry, properties, and commands that your device implements.
 
-### Get the required dependencies (Enivronmental Sensor)
-After cloning the Azure PnP Bridge repo to your machine, open the "Developer Command Prompt for VS 2017" and navigate to the directory of the cloned repo:
+If you haven't already downloaded the sample model files:
+
+1. Create a folder called models on your local machine.
+1. Right-click [Environmental Sensor](https://raw.githubusercontent.com/Azure/AzurePnPBridgePreview/pprupdatesforddtlv2/pnpbridge/docs/schemas/EnvironmentalSensorInline.capabilitymodel.json) and save the JSON file to the models folder.

```

Comment:
>https://raw.githubusercontent.com/Azure/AzurePnPBridgePreview/pprupdatesforddtlv2/pnpbridge/docs/schemas/EnvironmentalSensorInline.capabilitymodel.json [](start = 38, length = 151)

note this may change after we check in and this branch is deleted